app/javascript/sprocket-asset-import/editor/editor.js

```diff
@@ -1,4 +1,7 @@
-var CodeOceanEditor = {
+import * as Routes from 'generated/routes';
```

Comment:
I am not quite sure why Routes needs to be included here explicitly. I am a bit worried why this worked locally for me (now also failing for me).

We are setting `Routes` on the window in the `application.js` before requiring the editor. 
https://github.com/openHPI/codeocean/blob/main/app/javascript/application.js#L77 

This breaks my understanding of how JavaScript should work for me. Is Webpacker moving JS around?

Reply:
@christophblessing 
I agree that it would be better to implicitly import everything. In this transition, this is a bit difficult because this would mean a large code change. I want to first move the JavaScript sprockets and then refactor them with more and more implicit imports. 

@ibrahimkonuk 
Thank you for linking this article. I will look deeper into how webpack orders the objects.

I want to avoid some on of solution. Implicitly importing `Routes` here and using the window version in other places might work but will cause difficult issues down the line. I will keep this as a draft while researching.